docs/cmd/kn_service_create.md

```diff
@@ -53,48 +53,50 @@ kn service create NAME --image IMAGE
-      --no-lock-to-digest             Do not keep the running image for the service constant when not explicitly specifying the image. (--no-lock-to-digest pulls the image tag afresh with each new revision)
-      --no-wait                       Do not wait for 'service create' operation to be completed.
-  -p, --port string                   The port where application listens on, in the format 'NAME:PORT', where 'NAME' is optional. Examples: '--port h2c:8080' , '--port 8080'.
-      --pull-secret string            Image pull secret to set. An empty argument ("") clears the pull secret. The referenced secret must exist in the service's namespace.
-      --request strings               The resource requirement requests for this Service. For example, 'cpu=100m,memory=256Mi'. You can use this flag multiple times. To unset a resource request, append "-" to the resource name, e.g. '--request cpu-'.
-      --requests-cpu string           DEPRECATED: please use --request instead. The requested CPU (e.g., 250m).
-      --requests-memory string        DEPRECATED: please use --request instead. The requested memory (e.g., 64Mi).
-      --revision-name string          The revision name to set. Must start with the service name and a dash as a prefix. Empty revision name will result in the server generating a name for the revision. Accepts golang templates, allowing {{.Service}} for the service name, {{.Generation}} for the generation, and {{.Random [n]}} for n random consonants. (default "{{.Service}}-{{.Random 5}}-{{.Generation}}")
-      --scale int                     Minimum and maximum number of replicas.
-      --scale-init int                Initial number of replicas with which a service starts. Can be 0 or a positive integer.
-      --scale-max int                 Maximum number of replicas.
-      --scale-min int                 Minimum number of replicas.
-      --service-account string        Service account name to set. An empty argument ("") clears the service account. The referenced service account must exist in the service's namespace.
-      --user int                      The user ID to run the container (e.g., 1001).
-      --volume stringArray            Add a volume from a ConfigMap (prefix cm: or config-map:) or a Secret (prefix secret: or sc:). Example: --volume myvolume=cm:myconfigmap or --volume myvolume=secret:mysecret. You can use this flag multiple times. To unset a ConfigMap/Secret reference, append "-" to the name, e.g. --volume myvolume-.
-      --wait                          Wait for 'service create' operation to be completed. (default true)
-      --wait-timeout int              Seconds to wait before giving up on waiting for service to be ready. (default 600)
+  -a, --annotation stringArray            Service annotation to set. name=value; you may provide this flag any number of times to set multiple annotations. To unset, specify the annotation name followed by a "-" (e.g., name-).
```

Comment:
You should mention that this annotation is on Service *and* the Revision. Same like for label.